src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/NotificationServiceTest.java

```diff
@@ -124,10 +124,14 @@ void should_not_send_notification_when_notification_was_already_sent() {
 
         // when
         notificationService.sendNotifications(); //1st time
+        try {
+            new Thread().sleep(2000L);
```

Comment:
`Thread.sleep(2000)` or `TimeUnit.SECONDS.sleep(2)` second one is more readable. 
and no need to catch add to method signature exception

Reply:
ok